src/programs/project3d/project3d.cpp

```diff
@@ -225,13 +229,22 @@ bool Project3DApp::DoCalculation( ) {
     if ( project_based_on_star == true )
         number_of_projections_to_calculate = lines_to_process.GetCount( );
     else
+    {
```

Comment:
It seems like it has not been properly formatted using clang-format-14. I'll detail what should be seen below, however, code **must** be formatted properly using clang-format-14 prior to accepting any PR. 

Did you get this code directly from TG? If you open it in vscode in the dev container, add the brackets on the if branch (see below) then save it, it should be auto formated.

the else line should be 

```cpp
else {
// code
}
```

secondarily, returning to the alignment question, omitting brackets is okay on if/else when all branches are one line. Otherwise it is better to be consistent and have all branches use brackets - this makes bracket completion and "jump to next bracket" easier to use.

so 

```c++
if (project_based_on_star) {
//code
}
else {
// code
}
```

Reply:
also ... if (bool == true) is a more minor, but overly verbose expression. if (bool) is sufficient, except in some cases where if (nullptr == true) is needed.

Reply:
I did get this directly from Tim G, and did not think to run the formatter/just Ctrl + S the file for the auto-format. As to splitting lines, I will put another commit through that contains those changes (as well as the if (bool) rather than if (bool == true)).